lucien-ios/ComicDetailScreen/ComicDetailViewModel.swift

```diff
@@ -0,0 +1,72 @@
+//
+//  ComicDetailViewModel.swift
+//  lucien-ios
+//
+//  Created by Fang, Gracie on 10/24/17.
+//  Copyright © 2017 Intrepid Pursuits. All rights reserved.
+//
+
+import Foundation
+import RxCocoa
+import RxSwift
+
+final class ComicDetailViewModel {
+
+    private let dashboardComicUser: DashboardComicUser
+
+    let emptyDefault = "-"
+    let comicImage = Variable(UIImage())
+    lazy var comicTitle = Variable(emptyDefault)
+    lazy var storyTitle = Variable(emptyDefault)
+    lazy var volume = Variable(emptyDefault)
+    lazy var issue = Variable(emptyDefault)
+    lazy var releaseDate = Variable(emptyDefault)
+    lazy var publisher = Variable(emptyDefault)
+    lazy var genre = Variable(emptyDefault)
+    lazy var condition = Variable(emptyDefault)
+    lazy var borrowerName = Variable(emptyDefault)
+    lazy var dueDate = Variable(emptyDefault)
+
+    init(comic: DashboardComicUser) {
+        dashboardComicUser = comic
+        setObservables()
+    }
+
+    func setObservables() {
+        comicTitle.value = dashboardComicUser.dashboardComic.comicTitle
+        storyTitle.value = dashboardComicUser.dashboardComic.storyTitle
+        volume.value = dashboardComicUser.dashboardComic.volume ?? "-"
```

Comment:
What's the point of saving this as `emptyDefault` if you're not going to use it?